rotkehlchen/data_import/importers/bisq_trades.py

```diff
@@ -40,37 +41,23 @@ def _consume_bisq_trade(
         """
         if csv_row['Status'] == 'Canceled':
             return
-        timestamp = deserialize_timestamp_from_date(
-            date=csv_row['Date/Time'],
-            formatstr=timestamp_format,
-            location='Bisq',
-        )
+
         # Get assets and amounts sold
         offer = csv_row['Offer type'].split()
```

Comment:
```suggestion
        trade_type, base_asset_symbol = csv_row['Offer type'].split()
```
I think something like this would be an improvement here. Using `offer[x]` is a bit confusing.